docs/monitoring/tag-support.md

```diff
@@ -16,20 +16,30 @@ We recommend that you devise a set of tag keys that meet your needs for each pie
 
 Sematext supports 2 types of tags:
 
-1. **Physical tags** - Physical tag is an attribute of a data point one can use for filtering and grouping. They are sent with every data point. They are either automatically collected by agent or can be configured. e.g., hostname, jvm name, disk, elasticsearch index, tomcat webapp, port, etc. The maximum allowed length for key is 200 characters. The key should match this regex: `[a-zA-Z0-9_\-.:(\\ |,=)]+`.
-2. **Logical tags** - Logical tags are stored just once and updated periodically and are not sent as part of every data point. They are associated with a set of physical tags. One can filter/group data points using logical tags without sending them with every data point. For example, cloud tags like `env:prod` (on all production servers) and `env:test` (on all test servers) that you can define in AWS EC2 Console.  You can filter data in UI based on these tags. The maximum allowed length for both key and value is 1024 characters.
+1. **Physical tags** - Physical tag is an attribute of a data point one can use for filtering and grouping. They are sent with every data point. They are either automatically collected by agent or can be configured. e.g., hostname, jvm name, disk, elasticsearch index, tomcat webapp, port, etc. The maximum allowed length for key is 200 characters. The key should match this regex: `[a-zA-Z0-9_\-.:(\\ |,=)]+`. Physical tags are extracted from metric data sources and can be configured in metric definitions YAML file. 
+    * For example, refer to [Tomcat web module YAML definition](https://github.com/sematext/sematext-agent-integrations/blob/master/tomcat/jmx-web-module.yml) where the hostname and webapp name are extracted as physical tags from JMX ObjectName.
+    * Physical tags derived from metrics source can be omitted. In such cases, the data point will be aggregated on the omitted tag. By default, the aggregate function is used based on metric type (AVG for gauges and SUM for counters). This could be overridden using `agentAggregation` property of metric. Refer to [Elasticsearch index YAML definition](https://github.com/sematext/sematext-agent-integrations/blob/master/elasticsearch/json-index-0.yml) where `shard` tag is omitted.
+
+2. **Logical tags** - Logical tags are stored just once and updated periodically and are not sent as part of every data point. They are associated with a set of physical tags. One can filter/group data points using logical tags without sending them with every data point. Logical tags are either extracted automatically like cloud tags or user can specify Custom Tags. 
+    * For example, you can configure user-defined tags like `env:prod` (on all production servers) and `env:test` (on all test servers) in EC2 and filter the data in UI based on these tags. The maximum allowed length for both key and value is 1024 characters.
+
+`token` and `measurement` are reserved tag keys. Do not use them in user-defined tags.
 
 ## Cloud Tags
 
```

Comment:
Do we mention container labels we pick up as logical tags? Perhaps not the time yet since we don't show STA installation instructions for docker app?

Reply:
No, we don't. Yes, we can add them as part of STA docker release. Will add machine tags now.